src/sphinxexpr.cpp

```diff
@@ -2702,13 +2702,15 @@ class Expr_Case_c : public ISphStringExpr
 template<>
 void Expr_Case_c<true> :: DoCase ( char *pString ) const
 {
-    *pString = toupper(*pString);
+	// converts a letter into it's uppercase letter if exits.
+	*pString = toupper(*pString);
 }
 
 // For lower() function
 template<>
 void Expr_Case_c<false> :: DoCase ( char *pString ) const
 {
+	// converts a letter into it's lowercase letter if exists.
```

Comment:
also too clear for commenting. However if you think the comment IS necessary - at least consider to make concordant 'exits' and 'exists' in both your comments.